public/app/plugins/datasource/loki/querybuilder/operationUtils.ts

```diff
@@ -148,7 +148,7 @@ function operationWithRangeVectorRenderer(
 }
 
 export function labelFilterRenderer(model: QueryBuilderOperation, def: QueryBuilderOperationDef, innerExpr: string) {
-  if (model.params[1] === '<' || model.params[1] === '>') {
+  if (model.params[1] === '<' || model.params[1] === '>' || /\d/.test(model.params[2] as string)) {
```

Comment:
Then, I would discourage you from using 1) regexes and 2) type assertions:

```suggestion
  if (model.params[1] === '<' || model.params[1] === '>' || Number.isNaN(Number(model.params[2])) === false) {
```

Reply:
BTW, the `.isNaN()` is necessary because `NaN` is not false, or null, or undefined, it's just NaN.